### PR TITLE
Add to do list functionality

### PR DESCRIPTION
**Pull request by aliatayee**
## React To Do App
In this milestone i have developed the following parts:
- [ ] Configure react project structure
- [ ] Add input handle functionality
- [ ] Add local storage functionality
- [ ] Add task item lists
- [ ] Add edit functionality
- [ ] Add delete functionality
- [ ] Add styles

### CODE REVIEW

File: src/ClassBased/components/InputTodo.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react"
+
+class InputTodo extends Component {
+
+    state = {
+        title: ""
+      };
```
Comment (flower-v-codes) [inline src/ClassBased/components/InputTodo.js:7]: Your project works perfectly, but for best practices, it is advised to include a 'constructor' and a 'super' with 'props' as a parameter. And then your state can go inside the constructor and will look like this: this.state = {title: ""}, so to include the 'this' keyword in front of state.

File: src/ClassBased/components/InputTodo.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    return (
+      <form onSubmit={this.handleSubmit} className="form-container">
+        <input 
+        type="text" 
+        className="input-text"
+        placeholder="Add your task"
+        value={this.state.title}
+        name="title"
+        onChange={this.onChange} />
+        <button className="input-submit">Add</button>
```
Comment (flower-v-codes) [inline src/ClassBased/components/InputTodo.js:34]: For the button tags, it is advised to include a 'type' attribute, for example: type="button" or type="submit".

**Comment by aliatayee** [comment]
@virag-ky thank you very much for the feedbacks. 
I have made both class base and function base components and the function base component is configured for the final draft please ignore yhe class based forlder and review the function based

**Comment by flower-v-codes** [comment]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @aliatayee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

I left a few comments above which are optional, for best practices it's good to include those suggestions, and if you have linters, then it would warn you of these issues.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by flower-v-codes** [comment]
> @virag-ky thank you very much for the feedbacks. I have made both class base and function base components and the function base component is configured for the final draft please ignore yhe class based forlder and review the function based

Yes, the functional components are fine, the code is clean and organized, these were just some suggestions that I seen, otherwise everything is great with the code/project. 👍🏻

**Pull request merged**